.github/workflows/create-milestone.yml

```diff
@@ -61,6 +61,7 @@ jobs:
             "ansys/ansys-tools-path",
             "ansys/ansys-tools-protoc-helper",
             "ansys/ansys-tools-visualization-interface",
+            "ansys/pyadditive-widgets",
             "ansys/pyansys-tools-report",

```

Comment:
Follow alphabetical order

Reply:
Wouldn't pyadditive-widgets come before pyansys-tools since "pyad" is earlier in the alphabet than "pyan"?

Reply:
I did a comment edit sorry 😄